### PR TITLE
Config-file cleanup and code refactorisation 

### PR DESCRIPTION
**Pull request by AntonOlsson2**
Removes unnecessary data from the config files. Restructured createAtoms to only create atoms objects by the Atoms() constructor from ASE. This way testing and safety guards are easier to implement. We also only deal with primitive cells now, instead of sometimes having conventional representation and sometimes having primitive vectors. 

Later we may add hard-coded basis-matrices to bring back the user friendly "bravaislattice" to the config files.

Removed some old leftover prints from the main file. 

Removed all ASE and ASAP potentials, now we only use openKIM potentials. These are usually faster and more accurate. Also there exists a universal potential that works for most materials that can be added in as a default calculator later.

### CODE REVIEW

File: effective_velocity.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def longtitudinalSoundWaveVelocity(atom_list) :
     bulkModulus(atom_list) and density()"""
     #K = bulk_modulus(atom_list)
     K = 123 * 1E9 #bulk modulus for copper 
-    G = shear_modulus(atom_list)
+    #G = shear_modulus(atom_list)
```
Comment (martinclason) [inline effective_velocity.py:11]: Why is is the shear modulus commented out here?

File: main.py [hunk 2]
```diff
@@ -26,57 +26,19 @@ def MD(options):
...
-    sigma = options["sigma"] * units.Ang
-    cutoff = options["cutoff"] * units.Ang
     iterations = options["iterations"] if options["iterations"] else 200
     interval = options["interval"] if options["interval"] else 10
 
     if use_asap:
         print("Running with asap")
-        from asap3 import EMT
         from asap3.md.verlet import VelocityVerlet
-        from asap3 import LennardJones
```
Comment (martinclason) [inline main.py:43]: Looks like we need LennardJones pot as a good default if openKIM doesn't work. Maybe this should be added back now?

File: main.py [hunk 3]
```diff
@@ -26,57 +26,19 @@ def MD(options):
...
     else:
         print("Running with ase")
-        from ase.calculators.emt import EMT
-        from ase.calculators.lj import LennardJones
```
Comment (martinclason) [inline main.py:47]: ^LJ

File: main.py [hunk 4]
```diff
@@ -26,57 +26,19 @@ def MD(options):
...
         from ase.md.verlet import VelocityVerlet
 
-    def LJ(use_asap=use_asap):
```
Comment (martinclason) [inline main.py:50]: ^LJ

File: main.py [hunk 5]
```diff
@@ -26,57 +26,19 @@ def MD(options):
...
-                modified=True)
-        else:
-            return LennardJones(
-                epsilon=epsilon,
-                sigma=sigma)
-
     # Set up a crystal
     atoms = createAtoms(options)
-
-    def OpenKIMPotential():
```
Comment (martinclason) [inline main.py:66]: Maybe this should be kept to error handle openKIM more robustly?

File: main.py [hunk 6]
```diff
@@ -26,57 +26,19 @@ def MD(options):
...
-        try:
-            return KIM(options["openKIMid"])
-        except:
-            return None
-
-    known_potentials = {
```
Comment (martinclason) [inline main.py:72]: Could probably also be reused if keeping LJ.

File: test/config.yaml [hunk 7]
```diff
@@ -1,35 +1,23 @@
...
+latticeconstant : 10 #Only supporting cubic lattices with one parameter
 cell: [[0, 0.5, 0.5], [0.5, 0, 0.5], [0.5, 0.5, 0]] #(primitive unit cell vectors)
-positions : [[0,0,0]] #Set position of each atom in the molecule
+positions : [[0,0,0]] #Set position of each atom relative to each other in the molecule (lists in list)
 symbol : "Cu" 
 pbc : True #periodic boundary conditions
-size : [2,2,2]
-temperature_K : 600
-# Leave potential empty if running ASE, will resort to using EMT as potential.
-potential: "openKIM"
```
Comment (martinclason) [inline test/config.yaml:23]: Keep if keeping LJ^?

File: test/test_createAtoms.py [hunk 8]
```diff
@@ -1,12 +1,9 @@
-from createAtoms import createAtoms, createBravaislattice
+from createAtoms import createAtoms
 
 import yaml 
 
 config_file = open("test/config.yaml")
 parsed_config_file = yaml.load(config_file, Loader=yaml.FullLoader)
```
Comment (martinclason) [inline test/test_createAtoms.py:6]: Better to use config file parser module here instead of messing with yaml.

**Review by martinclason — CHANGES_REQUESTED** [review]
Should probably fix most of these before merging.

**Comment by martinclason** [comment]
Maybe good to make CI pass before merging this by defaulting to builtin LJ pot and not running open_kim related tests in CI.

File: effective_velocity.py (hunk 1 above)
Comment (martinclason) [reply effective_velocity.py]: Added todo to look at this later.

**Review by martinclason — APPROVED** [review]
